pkg/services/object/put/distributed.go

```diff
@@ -201,7 +180,7 @@ func (t *distributedTarget) Close() (oid.ID, error) {
 			t.metaSvc.NotifyObjectSuccess(objAccepted, addr)
 		}
 
-		err = t.cnrClient.SubmitObjectPut(t.objSharedMeta, t.collectedSignatures)
+		err = t.cnrClient.SubmitObjectPut(t.getShardObjectMetadata(), t.collectedSignatures)
```

Comment:
this was done this way cause i expected meta-on-chain to be the main option in our consistency problem. @roman-khimov, has smth changed?